plugin.yaml

```
@@ -1,31 +1,31 @@
 name: "aqua"
 repository: github.com/aquasecurity/trivy-plugin-aqua
-version: "v0.11.1"
+version: "v0.12.0"
 usage: trivy aqua <srcPath>
 description: A Trivy plugin that sends results to Aqua.
 platforms:
   - selector: # optional
       os: linux
       arch: amd64
-    uri: https://github.com/aquasecurity/trivy-plugin-aqua/releases/download/v0.11.1/linux_amd64_v0.11.1.tar.gz
+    uri: https://github.com/aquasecurity/trivy-plugin-aqua/releases/download/v0.12.0/linux_amd64_v0.12.0.tar.gz
     bin: ./aqua
   - selector:
       os: linux
       arch: arm64
-    uri: https://github.com/aquasecurity/trivy-plugin-aqua/releases/download/v0.11.1/linux_arm64_v0.11.1.tar.gz
+    uri: https://github.com/aquasecurity/trivy-plugin-aqua/releases/download/v0.12.0/linux_arm64_v0.12.0.tar.gz
     bin: ./aqua
   - selector:
       os: darwin
-      arch: amd64
-    uri: https://github.com/aquasecurity/trivy-plugin-aqua/releases/download/v0.11.1/darwin_amd64_v0.11.1.tar.gz
+      arch: arm64
```

```suggestion
      arch: amd64
```

This is fixed on the template here - https://github.com/aquasecurity/trivy-plugin-aqua/pull/43